### PR TITLE
refactor(core): Port event bus config (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
Follow-up to #10103

### CODE REVIEW

File: packages/cli/src/config/schema.ts [hunk 1]
```diff
@@ -679,21 +679,6 @@ export const schema = {
...
-			env: 'N8N_TEMPLATES_ENABLED',
-		},
-		host: {
-			doc: 'Endpoint host to retrieve workflow templates from endpoints.',
-			format: String,
-			default: 'https://api.n8n.io/api/',
-			env: 'N8N_TEMPLATES_HOST',
-		},
-	},
-
```
Comment (ivov) [inline packages/cli/src/config/schema.ts:696]: Forgot to remove in #10103

**Review by netroy — APPROVED** [review]

File: packages/@n8n/config/src/configs/event-bus.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+	/** Basename of event log file */
+	@Env('N8N_EVENTBUS_LOGWRITER_LOGBASENAME')
+	readonly logBaseName: string = 'n8nEventLog';
+}
+
+@Config
+export class EventBusConfig {
+	/** How often (in ms) to check for unsent event messages. Can in rare cases cause a message to be sent twice. `0` to disable */
+	@Env('N8N_EVENTBUS_CHECKUNSENTINTERVAL')
```
Comment (netroy) [inline packages/@n8n/config/src/configs/event-bus.ts:21]: should we rename this to `N8N_EVENTBUS_CHECK_UNSENT_INTERVAL` in v2?

Comment (ivov) [reply packages/@n8n/config/src/configs/event-bus.ts:21]: Let's please revisit and challenge as many env vars as reasonable in v2. Beyond naming, my impression is many of them are unused and should be removed. This one self-reports as buggy, which likely discourages even advanced users from using it.

**Comment by janober** [comment]
Got released with [`n8n@1.52.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.52.0)

**Pull request merged**